pkg/proto/hapi/chart/chart.pb.go

```diff
@@ -1,5 +1,6 @@
-// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I made sure I built this with libprotoc 3.2.0 but the comments are still different...